pkg/internal/codegen/parse/util.go

```diff
@@ -40,6 +40,8 @@ const (
 	printColumnFormat  = "format"
 	printColumnPri     = "priority"
 	printColumnError   = "invalid printcolumn path. name,type, and JSONPath are required kye-value pairs and rest of the fields are optinal. For example: // +kubebuilder:printcolumn:name=abc,type=string,JSONPath=status"
+	shortnamePrefix    = "+kubebuilder:shortname:"
```

Comment:
Instead of introducing new annotation, can we extend the existing `kubebuilder:resource` annotation to support an array ? we can take `;` delimited list. An example:

```Go
// kubebuilder:resource:path=services,shortName=ty;to;

```

Reply:
@droot yes, we can certainly extend that annotation instead of creating a new one. Let me know if you want me to file a PR for this?

Reply:
@magicsong you can probably change the code here
https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/internal/codegen/parse/index.go#L115
https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/internal/codegen/parse/index.go#L119
 to accommodate your logic.

Reply:
@sohankunkerkar i have changed the code, wdyt?

Reply:
@magicsong yeah, the changes look good to me.